src/test/scala/com/danielasfregola/zucchini/DictionaryTest.scala

```diff
@@ -9,12 +9,19 @@ class DictionaryTest extends WordSpecLike {
 
     "extractUniqueWordsFromTextFile" should {
 
-      "return 6 unique words" in {
-        assertResult(Set("rabbit", "lion", "pig", "horse", "Goat", "uniCorn23")){
+      "and return 6 unique words" in {
```

Comment:
super minor: why the "and"? I think it reads fine in English without :)

Reply:
Sure, will change it back in the next PR :)